configuration/scripts/cice.run.setup.csh

```diff
@@ -46,6 +46,7 @@ cd \${ICE_RUNDIR}
 setenv OMP_NUM_THREADS ${nthrds}
 
 cp -f \${ICE_CASEDIR}/ice_in \${ICE_RUNDIR}
+cp -f \${ICE_CASEDIR}/cice.settings \${ICE_RUNDIR}
```

Comment:
Minor point : in fact, I think *also* copying the env file could be a good idea in the context of #412, since then one could do from the run directory:
```bash
$ csh # if not already using csh
% source env.<machine>_<compiler>
% source cice.settings
% gdb ./cice  # or another debugger
```

This is a little bit easier than having to start csh in the case in the run directory, source both files, then `cd` to the run directory...

Reply:
Sounds good, I just checked that mod in.